core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -374,8 +374,9 @@ private void checkGraalVMVersion(String version) {
         final List<String> obsoleteGraalVmVersions = Arrays.asList("1.0.0", "19.0.", "19.1.", "19.2.", "19.3.0");
         final boolean vmVersionIsObsolete = obsoleteGraalVmVersions.stream().anyMatch(v -> version.contains(" " + v));
         if (vmVersionIsObsolete) {
-            throw new IllegalStateException(
-                    "Out of date version of GraalVM detected: " + version + ". Please upgrade to GraalVM 19.3.1.");
+            throw new IllegalStateException("Unsupported version of GraalVM detected: " + version + "."
```

Comment:
Please keep the out of date part as that's exactly what we test. People using 20.0.1 won't have this warning for instance.